transform/app/transform/live_update/training_ig.py

```diff
@@ -22,6 +30,29 @@ def process_message(frame) -> None:
     suspended = frame_body["suspended"]
     status = frame_body["status"]
 
+    collection, data, data_id = _extract_data_from_frame(raw_collection, frame_body)
+
+    logger.info(f"Received message, type: {raw_collection}, id: {data_id}")
```

Comment:
Received message -> Started to process message, type: {raw_collection}, id: {data_id}